_includes/footer.html

```diff
@@ -43,18 +48,53 @@
   integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
   crossorigin="anonymous"
 ></script>
+<script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/vue"></script>
 
 <script>
-  function on() {
-    document.getElementById("overlay").style.display = "block";
-  }
+    function on() {
+      document.getElementById("overlay").style.display = "block";
+    }
 
-  function off() {
-    document.getElementById("overlay").style.display = "none";
-  }
+    function off() {
+      document.getElementById("overlay").style.display = "none";
+    }
 
-  $(function() {
-    $('img').addClass('img-fluid');
-  })
+    $(function() {
+      $('img').addClass('img-fluid');
+    })
 
+    var cantstopcbus = new Vue({
```

Comment:
You're going to make me learn Vue aren't you?

Reply:
absolutely! :-)